libs/explorers/comparison-tools/src/lib/base-comparison-tool/base-comparison-tool.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { LoadingContainerComponent } from '@sagebionetworks/explorers/util';
+import { LoadingIconColors } from '@sagebionetworks/explorers/models';
+
+@Component({
+  selector: 'explorers-base-comparison-tool',
+  imports: [LoadingContainerComponent],
+  templateUrl: './base-comparison-tool.component.html',
+  styleUrls: ['./base-comparison-tool.component.scss'],
+})
+export class BaseComparisonToolComponent {
+  @Input() isLoading = true;
+  @Input() loadingIconColors!: LoadingIconColors;
```

Comment:
If we expect a value for `loadingIconColors`, should we make the input required? 

```suggestion
  @Input({ required: true }) loadingIconColors!: LoadingIconColors;
```